From 7a611b546ad4621c8b6a4645a7f9abdf529c1528 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Wed, 21 May 2025 19:08:38 +0300
Subject: [PATCH] [JSW-703] jgs: align with soc arch changes

With this patch, embedded specs aligned to use
new declarations from soc header.

Change-Id: Id75d31a8260b78cb13b515aefd00efe9485c97bd
---

diff --git a/fs/jgs/address_space.h b/fs/jgs/address_space.h
index 9f30f5a..3967616 100644
--- a/fs/jgs/address_space.h
+++ b/fs/jgs/address_space.h
@@ -37,10 +37,6 @@
  #define COMP0_MNG_ICCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_MNG_ARC_ICCM_BASE
  #define COMP0_MNG_DCCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_MNG_ARC_DCCM_BASE
 
- /* TODO: avoid using this hard coded macro, start using arc region macro from soc.h */
- #define LOCAL_ICCM_BASE			0
- #define LOCAL_DCCM_BASE			0x80000000
-
  /* 64b address used to convert specs 31bit address to full FS MEM address. Applicable to all dies */
  #define FW_MEM_FULL_BASE		0x300007F80000000ull
 
diff --git a/fs/jgs/soc.h b/fs/jgs/soc.h
index ea9650d..00a1a9a 100644
--- a/fs/jgs/soc.h
+++ b/fs/jgs/soc.h
@@ -49,9 +49,9 @@
 #define CONV_ARC_REG_TO_CFG_MEM(addr)	(((addr) & MASK_28_BIT) + ((CFG_MEM_FULL_BASE) & MASK_32_BIT))
 
 #define FLASH_VALID_MASK		(SZ_2M - 1)
-#define HL_FLASH_OFFSET			0xF000000
-#define HL_FLASH_SIZE			(16 * SIZE_MB)
-#define SPI_FLASH_BASE			(GEN_MEM_REG_BASE + HL_FLASH_OFFSET)
+#define SPI_FLASH_OFFSET		0xF000000
+#define SPI_FLASH_SIZE			(16 * SZ_1MB)
+#define SPI_FLASH_BASE			(GEN_MEM_REG_BASE + SPI_FLASH_OFFSET)
 
 #endif /* _SOC_H_ */
 
